manictime-client-plugin-example/source/tag-plugin/TagPlugin.Example/Settings/SettingsView.xaml

```diff
@@ -59,6 +61,24 @@
                 <Label Grid.Column="0" Content="Non-Billable Activity Id" VerticalContentAlignment="Center"/>
                 <TextBox Grid.Column="1" Text="{Binding NonBillableActivityId}" />
             </Grid>
+
+            <Grid Grid.Row="6" Margin="0 5">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="0.7*" />
+                    <ColumnDefinition/>
+                </Grid.ColumnDefinitions>
+                <Label Grid.Column="0" Content="WIQL Query" VerticalContentAlignment="Center"/>
+                <TextBox Grid.Column="1" Text="{Binding BillableWiqlQueryTemplate}" />
+            </Grid>
+
+            <Grid Grid.Row="7" Margin="0 5">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="0.7*" />
+                    <ColumnDefinition/>
+                </Grid.ColumnDefinitions>
+                <Label Grid.Column="0" Content="WIQL Query" VerticalContentAlignment="Center"/>
```

Comment:
These rows need the 'Content' attribute to be set. This appears as the label for the text box.